models/tests.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"github.com/go-xorm/core"
+	"github.com/go-xorm/xorm"
+	_ "github.com/mattn/go-sqlite3" // for the test engine
```

Comment:
In the future we can use build tags to allow choose what DB we will run test agains: MySQL/PG/SQLite/etc